string topTask
int32 reward